ironic-deployment/default/ironic_bmo_configmap.env

```diff
@@ -8,4 +8,4 @@ IRONIC_INSPECTOR_ENDPOINT=http://172.22.0.2:5050/v1/
 CACHEURL=http://172.22.0.1/images
 IRONIC_FAST_TRACK=true
 IRONIC_KERNEL_PARAMS=console=ttyS0
-INSPECTOR_REVERSE_PROXY_SETUP=false
+INSPECTOR_REVERSE_PROXY_SETUP=true
```

Comment:
why would this be true by default ? I would rather expect it to be false, and set to true via kustomization / replacement if deploying TLS. I know that the reverse proxy would not be deployed anyways, but having it set to true might be confusing in a non-reverse proxy scenario

Reply:
Fixed! Thanks